test/parallel/test-stream-finished.js

```diff
@@ -679,5 +679,6 @@ testClosed((opts) => new Writable({ write() {}, ...opts }));
 
   finished(stream, { readable: false }, common.mustCall((err) => {
     assert(!err);
+    assert.strictEqual(stream._writableState.pendingcb, 0);
```

Comment:
Can you add a test that does not look into _writableState? Why are you doing this change?

Reply:
sure, I will add a new test to address it, thanks!

> Why are you doing this change?

Do you mean the change to the test? If so, it is because when the callback function is called in `stream.finished` it won't wait for the callback in the `writable._write` (if any async action is performed) and one of the indication is `stream._writableState.pendingcb` is not 0.